eos-update-flatpak-repos

```diff
@@ -467,6 +467,556 @@ FLATPAKS_TO_MIGRATE = [
+        'old-branch': 'eos3',
+        'new-branch': 'stable',
+        'old-origin': 'eos-apps',
+        'new-origin': 'flathub'
+    },
+    {
+        'name': 'net.sourceforge.Btanks',
+        'new-name': 'net.sourceforge.btanks',
+        'kind': Flatpak.RefKind.APP,
+        'old-branch': 'eos3',
+        'new-branch': 'stable',
+        'old-origin': 'eos-apps',
+        'new-origin': 'flathub'
+    },
+    {
+        'name': 'net.sourceforge.ChromiumBSU',
+        'new-name': 'net.sourceforge.chromium-bsu',
+        'kind': Flatpak.RefKind.APP,
+        'old-branch': 'eos3',
+        'new-branch': 'stable',
+        'old-origin': 'eos-apps',
+        'new-origin': 'flathub'
+    },
+    {
+        'name': 'net.sourceforge.Extremetuxracer',
+        'new-name': 'net.sourceforge.ExtremeTuxRacer',
+        'kind': Flatpak.RefKind.APP,
+        'old-branch': 'eos3',
+        'new-branch': 'stable',
+        'old-origin': 'eos-apps',
+        'new-origin': 'flathub'
+    },
+    {
+        'name': 'net.sourceforge.Rili',
+        'new-name': 'net.sourceforge.Ri-li',
+        'kind': Flatpak.RefKind.APP,
+        'old-branch': 'eos3',
+        'new-branch': 'stable',
+        'old-origin': 'eos-apps',
+        'new-origin': 'flathub'
+    },
+    {
+        'name': 'net.sourceforge.Supertuxkart',
+        'new-name': 'net.supertuxkart.SuperTuxKart',
+        'kind': Flatpak.RefKind.APP,
+        'old-branch': 'eos3',
+        'new-branch': 'stable',
+        'old-origin': 'eos-apps',
+        'new-origin': 'flathub'
+    },
+    {
+        'name': 'net.sourceforge.Torcs',
+        'new-name': 'net.sourceforge.torcs',
+        'kind': Flatpak.RefKind.APP,
+        'old-branch': 'eos3',
+        'new-branch': 'stable',
+        'old-origin': 'eos-apps',
+        'new-origin': 'flathub'
+    },
+    {
+        'name': 'net.sourceforge.Tuxfootball',
+        'new-name': 'net.sourceforge.TuxFootball',
+        'kind': Flatpak.RefKind.APP,
+        'old-branch': 'eos3',
+        'new-branch': 'stable',
+        'old-origin': 'eos-apps',
+        'new-origin': 'flathub'
+    },
+    {
+        'name': 'net.sourceforge.Warmux',
+        'new-name': 'org.gna.Warmux',
+        'kind': Flatpak.RefKind.APP,
+        'old-branch': 'eos3',
+        'new-branch': 'stable',
+        'old-origin': 'eos-apps',
+        'new-origin': 'flathub'
+    },
+    {
+        'name': 'net.wz2100.Warzone2100',
+        'new-name': 'net.wz2100.wz2100',
+        'kind': Flatpak.RefKind.APP,
+        'old-branch': 'eos3',
+        'new-branch': 'stable',
+        'old-origin': 'eos-apps',
+        'new-origin': 'flathub'
+    },
+    {
+        'name': 'org.armagetronad.Armagetronad',
+        'new-name': 'org.armagetronad.ArmagetronAdvanced',
+        'kind': Flatpak.RefKind.APP,
+        'old-branch': 'eos3',
+        'new-branch': 'stable',
+        'old-origin': 'eos-apps',
+        'new-origin': 'flathub'
+    },
+    {
+        'name': 'org.codeblocks.App',
```

Comment:
As commented elsewhere, this app's manifest defines a Debug and a Locale extension, but neither appears to be present in the `eos-apps` repository, so no great loss that we don't mangle those extensions in the manifest.

Reply:
I can only assume they were released to eos-apps at a time that the release-apps script was blissfully unaware of app extensions. Luckily for us... :)